src/js/utils/add-click-action.ts

```diff
@@ -0,0 +1,10 @@
+import UI from 'utils/ui';
+
+export function addClickAction(element: HTMLElement, clickAction: (evt) => void, ctx?: any): UI {
+    const ui = new UI(element, { directSelect: true });
+    ui.on('click tap enter', function (evt) {
+        clickAction(evt);
+    }, ctx);
+    
+    return ui;
+}
```

Comment:
I would expect this module to go in /view/utils as opposed to /utils since it is only used by other components of /view. Can you move it?